### DISCUSSION TITLE
Hamburger menu on the right side

### DISCUSSION THREAD

**Discussion by Janoo1606** [templaza/astroid-framework#276] [Q&A] [answered]
Hello,
is it possible to place the hamburger menu on the right side in mobile view?

**Comment by Chacapamac** [comment, marked as answer]
I @Janoo1606 ,
Happy New Year!

For now, the only way I see
```
div.header-mobilemenu-trigger {
position: absolute;
 right: 20px;
}
```

I will try to find something more elegant.

**Reply by Janoo1606** [reply]
Thank you. 
A simple button in the astroid settings would be nice ;)

**Reply by Chacapamac** [reply]
Yes, I agree with you. I mark this as a Enhancement

**Comment by ginageick** [comment]
Is there an update on this?

**Comment by JohnyJanek** [comment]
Hi, it would be useful for me too :-) Is there anything happening on this topic?
